vinyl/Vinyl_2022-08-04.sql

```diff
@@ -0,0 +1,70 @@
+# ************************************************************
```

Comment:
having the date on the DB means you end up with lots of versions of the DB in your repo. there should only be one DB file that should always have the same name so I can do a diff on it